evansmusomi/burger-builder
# Refactors Reducer Cases for Readability

*What this does*
- Encapsulates reducer case logic into functions to make switch statement easier to follow

*How to verify*
- A user can build a burger and order as usual
- A developer can verify that the reducer switch statements are easier to follow in the source code
